src/actions/background/auto.ts

```diff
@@ -0,0 +1,11 @@
+import BackgroundAction from "./BackgroundAction";
+import {IBackgroundAction} from "./IBackgroundAction";
+
+/**
+ * Applies background color automatically.
+ */
+function auto(): IBackgroundAction{
```

Comment:
Functions without a namespace applied to them (in the docstring) will appear as public in the JSdoc,
This may or may not be what you wanted to implement.